specs/Qowaiv.Specs/Chemistry/CAS_Registry_Number_specs.cs

```diff
@@ -0,0 +1,489 @@
+﻿using Qowaiv.Chemistry;
+
+namespace Chemistry.CAS_Registry_Number_specs;
+
+public class With_domain_logic
+{
+    [TestCase("")]
+    [TestCase("?")]
+    public void has_length_zero_for_empty_and_unknown(CasRegistryNumber svo)
+        => svo.Length.Should().Be(0);
+
+    
+    [TestCase(5, "73–24–5")]
+    [TestCase(7, "7732-18-5")]
+    [TestCase(8, "10028-14-5")]
```

Comment:
Testcase with length 10?

Reply:
I chose to use numbers that are have been assigned and are publicly available. I couldn't find a public one that is.